markdown/chapters/meandeck/alternative-piles.md

```diff
@@ -0,0 +1,203 @@
+---
+authors: Doishy
+title: Alternative Pile Building
+---
+
+## Contents
+
+## Preamble
+
+In this chapter we will present some alternative piles which may be utilised
+depending on the tools available in your deck. This could include spicy main
+deck choices or the use of certain sideboard cards and is here to make you aware
+of how some of these tools are utilised in the wild. These are not listed in
+any specific order.
+
+## Shelldock Isle Piles
+
+You might see a lot of people running card:[SI] alongside card:[Em] in their 
+sideboards. The purpose of these cards is to use them in a pile that is 
+effective against blue based control decks which do not run cards like
+Wasteland. This pile often requires passing multiple turns and can be used
+in conjunction with Oracle. 
+
+> PTT
+
+:::row{variant=pile}
+- SI
+- Em
+- Island
+- CoS
+- TO
+:::
+
+In theory you can replace Emrakul with any card (including lands) however
+the uncounterable nature, the take an extra turn on casting (which still 
+works with Shelldock!) and the attack for 15 with Annihilator 6 all make it
+probably the most effective tool to use. 
+
+## Piles using Dragon's Rage Channeler
+
+Some people have tried various creature shells alongside the Doomsday package
+and one of the more interesting ones for use as part of the Doomsday
+resolution is card:[Dragon's Rage Channeler] (DRC). Each spell cast can let you "dig"
+one additional card deeper through your pile thanks to the Survail ability.
+Let's look at a couple of examples.
+
+> DRC in play + non-Consider castable cantrip in hand
+
+:::row{variant=pile}
+- DA
+- LED
+- SW
+- LP
+- TO
+:::
+
+The most basic example above allows you to turn any regular cantrip into a 
+pseudo copy of card:[Con] by giving it a Surveil 1 effect attached. This allows
+you to perform the basic Consider pile using either card:[Pnd], card:[Pre] or
+card:[BS].
+
+> {U} + DRC in play + non-Brainstorm castable cantrip in hand + X
+
+:::row{variant=pile}
+- SW
+- BS
+- LED
+- EoA
+- TO
+:::
+
+Here you can execute a Brainstorm pile even if you are at a low life total without the
+use of card:[SW] or if you don't have access to tools like card:[IU] or card:[Con] which
+still allows you to get to 0 cards left in the deck. You can follow the listed steps:
```

Comment:
```suggestion
still allows you to get to 0 card left in the deck. You can follow the listed steps:
```

Reply:
Not sure why this change suggested, should be "cards"